tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.Repository/UserRepository/UserRepository.cs

```diff
@@ -36,7 +36,15 @@ public class UserRepository:IUserRepository
         /// <returns></returns>
         public Users GetUserProfilePicture(Client client)
         {
-            return userDetails.GetUserProfilePicture(client).Result;
+            try
```

Comment:
For profile picture, we are getting multi factor authentication issue. To solve that issue, we are throwing the exception